models/donation.model.js

```diff
@@ -1,16 +1,25 @@
-const mongooose = require('mongoose');
+const mongoose = require('mongoose');
 
 const DonationSchema = new mongooose.Schema({
-  donor: { type: mongooose.ObjectId, required: true },
-  ngo: { type: mongooose.ObjectId, required: true },
-  isDelivered: { type: Boolean, default: false },
+  user: { type: mongoose.Schema.Types.ObjectId, ref: "User" },
+  ngo: { type: mongoose.Schema.Types.ObjectId, ref: "NGOs" },
+  location: {
+    address: { type: String },
+    city: { type: String },
+    pin_code: { type: Number },
+    lattitude: { type: String },
+    longitude: { type: String },
+  },
```

Comment:
In location address, city, etc shouldn't be necessary fields?

Reply:
@PragsJ will do